Integrations/integration-Giphy.yml

```diff
@@ -0,0 +1,80 @@
+description: Display random GIF in the War Room (e.g. !giphy hello)
+detaileddescription: 'You can get API Key from: https://github.com/Giphy/GiphyAPI#access-and-api-keys'
+configuration:
+- display: Giphy URL
+  name: url
+  defaultvalue: http://api.giphy.com/v1/gifs/random
+  type: 0
+  required: true
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+script:
+  script: |-
+    var sendRequest = function(tag) {
+        var serverUrl = params.url;
+        var apiKey = params.apikey;
+        var requestUrl = params.url + '?api_key=' + apiKey + '&tag=' + tag;
```

Comment:
you need to sanitize the tag for special characters 
We have a function for this